ext/pg_connection.c

```diff
@@ -4115,6 +4153,10 @@ init_pg_connection()
 	sym_static_symbol = ID2SYM(rb_intern("static_symbol"));
 
 	rb_cPGconn = rb_define_class_under( rb_mPG, "Connection", rb_cObject );
+/* Help rdoc to known the Constants module */
+#if 0
+	rb_mPGconstants = rb_define_module_under( rb_mPG, "Constants" );
```

Comment:
Interesting. Does rdoc not notice this same call in `ext/pg.c`?

Reply:
Our current documentation [about PG::Connection](https://deveiate.org/code/pg/PG/Connection.html) doesn't resolve the include module but prints "Included Modules rb_mPGconstants" instead. This is fixed by this trick.